.github/workflows/oasislmf-unittest.yml

```
@@ -0,0 +1,50 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        cfg:
+        - { python-version: '3.7', pkg-version: ""}
+        - { python-version: '3.8', pkg-version: ""}
+        - { python-version: '3.9', pkg-version: ""}
+        - { python-version: '3.10', pkg-version: ""}
+        - { python-version: '3.10', pkg-version: 'numba==0.55.1' }
```

Now that this CI works, I'd add further tests to the matrix for better coverage
```suggestion
        - { python-version: '3.7', pkg-version: ""}
        - { python-version: '3.7', pkg-version: 'numba==0.55.1' }
        - { python-version: '3.8', pkg-version: ""}
        - { python-version: '3.8', pkg-version: 'numba==0.55.1' }
        - { python-version: '3.9', pkg-version: ""}
        - { python-version: '3.9', pkg-version: 'numba==0.55.1' }
        - { python-version: '3.10', pkg-version: ""}
        - { python-version: '3.10', pkg-version: 'numba==0.55.1' }
```

---

I think lets leave this as is for the moment, it feels enough on a per commit basis. We could add a more complete set of checks on the master / main branch only